app/mailers/notifier.rb

```diff
@@ -16,7 +16,7 @@ def notify(notification)
     @reply_link = reply_link(@notification)
 
     # Encrypting
-    @signed_message = verifier.generate ({ member_id: @notification.recipient.id, type: :send_notification_email })
+    @signed_message = verifier.generate(member_id: @notification.recipient.id, type: :send_notification_email)
```

Comment:
So, i need I'm really a python dev who pretends to know ruby.
Rubocop complained about the {} being unnecessary, hinting that ruby treats it as a hash regardless. I took it at it's word.